war/src/main/js/plugin-manager-ui.js

```diff
@@ -59,6 +59,10 @@ var handleFilter = function (e) {
 var debouncedFilter = debounce(handleFilter, 150);
 
 document.addEventListener("DOMContentLoaded", function () {
+  document
+    .querySelector("#side-panel")
+    .classList.add("app-page-body__sidebar--sticky");
```

Comment:
shouldn't this be a feature of the sidebar? Is there a reason you aren't adding it in jelly?

Reply:
Yeah it should be, was a workaround until this https://github.com/jenkinsci/jenkins/pull/8269 is implemented. I can remove it from this branch.